packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1521,4 +1527,104 @@ void main() {
     expect((mergeableMaterial.children.first as MaterialSlice).color, firstPanelColor);
     expect((mergeableMaterial.children.last as MaterialSlice).color, secondPanelColor);
   });
+
+  testWidgets('ExpansionPanelList expanded color test', (WidgetTester tester) async {
+    const Color expandedColor = Colors.orange;
+    const Color color = Colors.red;
+
+    const Key firstPanelKey = Key('firstPanelKey');
+    const Key secondPanelKey = Key('secondPanelKey');
```

Comment:
Since we can find the `IconButton` without needing the key (above conversation), you can remove them.